### ISSUE TITLE
GPU usage is 0

### ISSUE THREAD

**Issue by pijiawei** [vec2text/vec2text#78]
Hello! Sorry to bother you.
During the running process, I found that the running was very slow, but the GPU usage was always 0. Is this normal?
![image](https://github.com/user-attachments/assets/670ea599-4b57-44a9-8067-ed62d84f7d47)

**Comment by jxmorris12** [comment]
No, this isn't normal. :) You're just running invert_strings, is that correct?

What does this code output?
```python
from vec2text.models.model_utils import device
print(device)
```
